cps/CP4/CP21.md

```diff
@@ -0,0 +1,51 @@
+---
+cp: 21
+title: Updating Tinlake Rewards Allocation
+authors: Governance and Coordination Group (ImdioR, Rhano)
+contributors:
+proposal-type: CP-4 technical
+date-proposed: 2022-12-16
+date-passed: 
+status: polling
+---
+
+## Short Summary
+Renewal of incentive rewards for Tinlake
+
+## High-level objective
+This proposal is for minting 1,800,000 CFG into the Tinlake rewards account. Keeping the same rewards rate for TIN,DROP, AO as before.
+
+## Background
+To date Total Rewards 23,821,073 CFG has been awarded to both AOs and LPs on Tinlake, currently, CFG 628,868 is available to be distributed to Tinlake investors, which means we are nearing the end of the current rewards allocation. With the current TIN/DROP Rewards Rate and TVL, these awards will last approximately for the next 35 days.
+
+Despite the fact that Tinlake TVL decreased in the last 90 days, we would like to propose minting the same amount of CFG rewards. There are a few reasons for this:
+
+* The possibility of onboarding new pools. We would like to keep a small margin in order to cover any new onboarding or fast-increasing Tinlake TVL
+* Avoid reward deficit and guarantee regular rewards claiming for Tinlake investors. 
+* Taking the governance process (off-chain and on-chain voting) into consideration
+
+## Detailed description of the proposal:
+
+We would like to propose to mint and allocate an additional 1.8M CFG towards Tinlake liquidity rewards and keep the same rewards rate for TIN/DROP/AO [Council motion 18](https://gov.centrifuge.io/t/council-motion-18-tinlake-lp-rewards-restructuring-july-2022/4400)
+
+Approximatively an additional minting of 1.8M CFG with the current actual Rewards Rate and TVL will cover the upcoming rewards for the next 100 days from one side and will permit keeping inflation under 2.5% and remain responsive to any change to the external market conditions from another side.
+
+### Proposal Parameters:
+The proposal being put forward is the following:
+```
+1. Mint 1’800’000 CFG into the rewards account
+2. Keep the same rewards rate for TIN/DROP at 0.0003 CFG

```

Comment:
I think we should specify the units. It is 0.0003 CFG/DAI per day.

Reply:
2. Keep the same rewards rate for TIN/DROP at 0.0003 CFG/DAI

Here?